api/Events/Auth/Client/ClientWantsToLogIn.cs

```diff
@@ -3,13 +3,15 @@
 using Core.Services;
 using Fleck;
 using lib;
+using Shared.Dtos;
 using Shared.Dtos.FromClient;
 using Shared.Dtos.FromClient.Identity;
 using Shared.Exceptions;
+using Shared.Models;
 
 namespace api.Events.Auth.Client;
 
-public class ClientWantsToLogInDto : BaseDto
+public class ClientWantsToLogInDto : BaseDtoWithJwt
```

Comment:
the login event cannot require a JWT, as the purpose of logging in is to get a token